index.js

```diff
@@ -13,10 +13,19 @@ try {
 
 var API_PATH = '/api/v0/'
 
-module.exports = function (host, port) {
-  if (!host) host = 'localhost'
+module.exports = function (host_or_multiaddr, port) {
+  var host
+
+  if (!host_or_multiaddr) host = 'localhost'
   if (!port) port = 5001
 
+  if (host_or_multiaddr.match('/')) {
+    // is a multiaddr
+    var parts = host_or_multiaddr.split('/')
```

Comment:
Could probably use the multiaddr package here instead.

https://github.com/jbenet/node-multiaddr